pandas/io/formats/style_render.py

```diff
@@ -771,10 +788,13 @@ def wrapper(x):
     return wrapper
 
 
-def _str_escape_html(x):
-    """if escaping html: only use on str, else return input"""
+def _str_escape(x, escape):
+    """if escaping: only use on str, else return input"""
     if isinstance(x, str):
-        return escape_html(x)
+        if escape == "html":
+            return escape_html(x)
+        elif escape == "latex":
+            return _escape_latex(x)
```

Comment:
if it's anything else (not None) it's an error

pls add a test

Reply:
done